Practice2/src/partials/bot-section.html

```diff
@@ -15,32 +15,32 @@ <h5 class="bot-main-heading">Hofmann</h5>
   </div>
   <div class="bot-container-secondary">
     <ul class="bot-text-list-sub bot-one">
-      <li class="text-list">About Us</li>
-      <li class="text-list">Explore</li>
-      <li class="text-list">Contact</li>
+      <a href="javascript:void(0)" class="link-success"><li class="text-list">About Us</li></a>
```

Comment:
a tag should not cover li. li will cover a tag.

Reply:
Fixed